mhubert-pmn/Back-Url-Shortener
# [DevOps] Dockerize back

## Task
Dockerize Back

## Type of change

## Description
- Add docker-compose files
- Initialized back

## Testing scenario
In the terminal (you have to be in the back-url-shortener folder):
- Launch and build: `docker-compose up`
- see database at http://localhost:8301/ with phpmyadmin
- Down and delete running containers, volumes and networks: `docker-compose down`

## Trello card
[https://trello.com/c/0WYAxxfO](https://trello.com/c/0WYAxxfO)
